contracts/oracles/lido/LidoOracle.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.6;
+
+import '@yield-protocol/utils-v2/contracts/access/AccessControl.sol';
+import '@yield-protocol/utils-v2/contracts/cast/CastBytes32Bytes6.sol';
+import '@yield-protocol/utils-v2/contracts/token/IERC20Metadata.sol';
+import '@yield-protocol/vault-interfaces/IOracle.sol';
+import '../../constants/Constants.sol';
+import './IWstETH.sol';
+
+/**
+ * @title LidoOracle
+ * @notice
+ */
+contract LidoOracle is IOracle, AccessControl, Constants {
```

Comment:
in general the natspec on this contract is a little light, although I realize there probably isn't much explanation necessary and that its based on other Yield oracles

`T6 - Is the purpose of the contract and how it interacts with others documented using natspec?`
`T12 - Summarize the purpose and functionality of the contract with a @notice natspec comment. Document how the contract interacts with other contracts inside/outside the project in a @dev natspec comment.`
`V5 - Is the purpose of the variable and other important information documented using natspec?`
`F15 - Are all arguments, return values, side effects and other information documented using natspec?`

Reply:
I'm never going to say no to good natspec.